packages/contracts/deploy/014-OVM_L1MultiMessageRelayer.deploy.ts

```diff
@@ -18,6 +19,20 @@ const deployFn: DeployFunction = async (hre) => {
     name: 'OVM_L1MultiMessageRelayer',
     args: [Lib_AddressManager.address],
   })
+
+  // OVM_L2MessageRelayer *must* be set to multi message relayer address on mainnet.
+  if (hre.network.name.includes('mainnet')) {
+    const OVM_L1MultiMessageRelayer = await getDeployedContract(
```

Comment:
Does this method throw if the given string is not set in the address manager?  We should make sure we explicitly avoid a situation where we end up calling `setAddress`, but the input is still `0x00...`, so if it does not, we should add the check here.